src/cartridges/int_adyen_controllers/cartridge/templates/default/adyenform.isml

```diff
@@ -0,0 +1,22 @@
+<!--- TEMPLATENAME: 3dform.isml --->
+<isdecorate template="checkout/pt_checkout">
+	<iscontent type="text/html" charset="UTF-8" compact="true"/>
+		<isinclude template="util/modules"/>
+
+		<isinclude template="util/reporting/ReportCheckout.isml"/>
+		<div class="checkout">
+			<div class="payerauth">
+				<form action="${pdict.issuerUrl}" method="post" id="3dform" name="3dform">
+					<input type="hidden" name="TermUrl" value="${URLUtils.httpsContinue()}">
+					<input type="hidden" name="PaReq" value="${pdict.paRequest}">
+					<input type="hidden" name="MD" value="${pdict.md}">
+				</form>
+				<script language="javascript">
+					function load(){
+						document.getElementById('3dform').submit();
+					}
+					load();
+				</script>
+			</div>
+		</div>
+</isdecorate>
```

Comment:
why is this file added ?

Reply:
This file is moved from overlay to controllers